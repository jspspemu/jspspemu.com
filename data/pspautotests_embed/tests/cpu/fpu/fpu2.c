/*
FPU Test. Originally from jpcsp project:
http://code.google.com/p/jpcsp/source/browse/trunk/demos/src/fputest/main.c
Modified to perform automated tests.
*/

#include <common.h>
#include <math.h>

#include <pspkernel.h>

float floatRelevantValues[] = {
	 -20000.5
};

int main(int argc, char *argv[]) {
	//asm volatile("dbreak\n" : : );
	//printf("FPU2 Test:\n");
	printf("%f\n", -20000.5);
	printf("%f\n", floatRelevantValues[0]);
	return 0;
}
