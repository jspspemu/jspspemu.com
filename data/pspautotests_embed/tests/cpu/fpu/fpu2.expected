-20000.500000
-20000.500000
